pkg/plugin/plugin.go

```diff
@@ -84,10 +94,33 @@ func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificat
 		}, fmt.Errorf("failed to validate justification: %w", err)
 	}
 
+	if len(result.Matches) == 0 || len(result.Matches[0].MatchedIssues) == 0 {
+		return nil, fmt.Errorf("failed to get the matched jira issue %q", req.Justification.Value)
+	}
+
+	// There is only one JQL and one issueKey, so the first match result is
+	// checked directly.
+	if len(result.Matches[0].MatchedIssues) == 1 {
+		issueID := strconv.Itoa(result.Matches[0].MatchedIssues[0])
+
+		// The format for the Jira issue URL follows the pattern "thttps://<user’s subdomain>.atlassian.net/browse/<issueKey>".
```

Comment:
```suggestion
		// The format for the Jira issue URL follows the pattern "https://your-domain.atlassian.net/browse/<issueKey>".
```
Do you have any official jira reference doc?